server/db/localDB/tablesConfig.js

```diff
@@ -0,0 +1,19 @@
+const Pool = require('pg').Pool
+const pool = new Pool({
+    user: 'admin',
+    host: 'localhost',
+    database: 'spp',
+    password: 'password',
+    port: 5432,
+})
+
+const fs = require('fs');
+const path = require("path");
+
+const tablesSql = fs.readFileSync(path.resolve(__dirname, "tables.sql")).toString();
+
+const configurateDatabase = pool.query(tablesSql)
```

Comment:
I guess this piece of code doesn't work as expected since configureDatabase is a const field, so it's initialized with value when interpreter reads file, and that's why there is no need to access .configurateDatabase variable. I may be wrong.

Reply:
Please make it a function and add function for inserting test data:

`function configureDatabase() {
    const tablesSql = fs.readFileSync(path.resolve(__dirname, "tables.sql")).toString();
    pool.query(tablesSql)
}

function addTestData() {
    const testDataSql = fs.readFileSync(path.resolve(__dirname, "testData.sql")).toString();
    pool.query(testDataSql)
}

module.exports = {
    configureDatabase,
    addTestData
}`